idea_town/frontend/static-src/app/lib/page-manager.js

```diff
@@ -37,6 +41,16 @@ export default class PageManager {
       const Ctor = this.pages[page];
       const newPage = new Ctor(opts);
 
+      if (newPage.headerScroll) {
```

Comment:
Alternatively, we can have the "page" views live in a wrapper "app" view that owns the header directly and swaps main content with the page switcher. [I did something like that with my C25k app this summer that manages visibility of back & help buttons via CSS based on the current view](https://github.com/lmorchard/c25k-web/blob/master/src/js/views/App.js).